src/shared/components/GenericList/GenericList.js

```diff
@@ -210,55 +211,54 @@ export const GenericList = ({
     ));
   };
 
-  const tableClassNames = classnames(
-    'fd-table',
-    'fd-table--no-horizontal-borders',
-    { compact },
-  );
   const panelClassNames = classnames(
-    'generic-list',
+    'ui5-generic-list',
     {
       'fd-margin--md': !className?.includes('fd-margin'),
     },
     className,
   );
 
+  const tableClassNames = classnames('ui5-generic-list', {
+    'ui5-content-density-compact': compact,
+  });
+
   return (
-    <LayoutPanel className={panelClassNames} data-testid={testid}>
-      <LayoutPanel.Header className="fd-has-padding-left-small fd-has-padding-right-small">
-        <LayoutPanel.Head title={title} />
-        <LayoutPanel.Actions>{headerActions}</LayoutPanel.Actions>
-      </LayoutPanel.Header>
+    <>
```

Comment:
Please remove that, it will cause more conflicts with ui5-panel PR and doesn't need to be here.
```suggestion
```